core/src/main/java/nu/yona/server/subscriptions/service/PinResetRequestService.java

```diff
@@ -66,7 +71,7 @@ public void requestPinReset(UUID userId)
 	@Transactional
 	public void sendPinResetConfirmationCode(UUID userId)
 	{
-		User user = userService.getUserEntityById(userId);
+		User user = userService.lockUserForUpdate(userId);
```

Comment:
I don't get it. Does this just silently proceed when the user is removed?

Reply:
See my other comment.